packages/flutter/test/cupertino/segmented_control_test.dart

```diff
@@ -48,6 +48,38 @@ StatefulBuilder setupSimpleSegmentedControl() {
   );
 }
 
+StatefulBuilder buildOneDisabledSegmentedControl({required int disabledIndex}) {
```

Comment:
Should this parameter type be a set? Currently the `disabledChildren` set will only contain one child.

Reply:
As mentioned above, I removed `buildOneDisabledSegmentedControl`.